TOOLS/gen-x11-icon.sh

```diff
@@ -4,8 +4,8 @@
 # etc/mpv-icon...), and it will write video/out/x11_icon.bin.
 
 conv() {
-    echo
-    echo `identify -format "icon: %w %h" "$1"`
+    printf "\n"
```

Comment:
Why not just pre-append the `\n` to the next line if you're converting the echos to printfs?